src/Microsoft.ML.LightGbm/LightGbmMulticlassTrainer.cs

```diff
@@ -75,6 +75,12 @@ public enum EvaluateMetricType
                 LogLoss,
             }
 
+            /// <summary>
+            /// Whether training data is unbalanced.
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Use for binary classification when training data is not balanced.", ShortName = "us")]
```

Comment:
> Use for binary classification when training data is not balanced.

It's odd to say "use for binary classification" when this is the multiclass trainer.

Reply:
Is it more helpful to just say "Use for multi-class classification when training data is not balanced."